tests/integrationv2/README.md

```diff
@@ -127,6 +127,15 @@ In each provider that supports your feature you need to check if the flag is set
 line option for that particular provider. You can also add logic checks, e.g with client authentication
 the client must have a certificate to send. Otherwise the test will fail.
 
+## Fine-tune how input data is sent in the tests
+
+There are several optional arguments in the managed_process function. Use these arguments to have a
```

Comment:
Can we list these options with their description/usage? This would make it easier to read.